src/components/CohortBuilder/SqonBuilder/index.js

```diff
@@ -97,9 +97,9 @@ const SqonBuilder = compose(
               api={api}
               arrangerProjectId={arrangerProjectId}
               arrangerProjectIndex={ARRANGER_API_PARTICIPANT_INDEX_NAME}
-              FieldOpModifierContainer={props => {
-                return <FieldFilterContainer className="" showHeader={false} {...props} />;
-              }}
+              FieldOpModifierContainer={(props) => (
+                <FieldFilterContainer className="" showHeader={false} {...props} />
```

Comment:
if possible (I know it's risky) could we try to remove ...props and ...rest or at least expose as many props as possible?